contracts/mainnet/connectors/compound-iii/helpers.sol

```diff
@@ -0,0 +1,148 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.7.0;
+pragma abicoder v2;
+
+import { TokenInterface } from "../../common/interfaces.sol";
+import { DSMath } from "../../common/math.sol";
+import { Basic } from "../../common/basic.sol";
+import { CometInterface } from "./interface.sol";
+
+abstract contract Helpers is DSMath, Basic {
+	struct BorrowWithdrawParams {
+		address market;
+		address token;
+		address from;
+		address to;
+		uint256 amt;
+		uint256 getId;
+		uint256 setId;
+	}
+
+	function getBaseToken(address market)
+		internal
+		view
+		returns (address baseToken)
+	{
+		baseToken = CometInterface(market).baseToken();
+	}
+
+	function _withdraw(
+		address market,
+		address token,
+		address from,
+		address to,
+		uint256 amt
+	) internal {
+		if (from == address(0) && to == address(0)) {
+			CometInterface(market).withdraw(token, amt);
+		} else if (from == address(0)) {
+			CometInterface(market).withdrawTo(to, token, amt);
+		} else if (from != address(0) && to != address(0)) {
+			CometInterface(market).withdrawFrom(from, to, token, amt);
+		}
+	}
+
+	function _transfer(
+		address market,
+		address token,
+		address from,
+		address to,
+		uint256 amt
+	) internal {
+		bytes memory data;
```

Comment:
@Richa-iitr This is not getting used right?

Reply:
No